src/olympia/constants/base.py

```diff
@@ -415,12 +415,10 @@
 
 # Types of Canned Responses for reviewer tools.
 CANNED_RESPONSE_ADDON = 1
-CANNED_RESPONSE_APP = 2
 CANNED_RESPONSE_PERSONA = 3
 
 CANNED_RESPONSE_CHOICES = {
     CANNED_RESPONSE_ADDON: _('Add-on'),
-    CANNED_RESPONSE_APP: _('App'),
```

Comment:
This is just cleanup. `CANNED_RESPONSE_APP` was only used in that one test and not in actual code.